test/qa/lib/meshnet.py

```diff
@@ -84,37 +84,78 @@ def get_peers(output: str) -> list:
     return peers
 
 
-def get_this_device(output: str):
-    """Parses current device hostname from 'nordvpn meshnet peer list' output."""
+def peer_list_search(output:str, look_after_keyword: str, looking_for: str):
```

Comment:
I think a lot of different things can be thought of or done, but we should probably have a discussion with the team, towards what direction do we want to move with these tests, and if we plan to have multiple peers later on, etc.

On the other hand, maybe it would make sense to make code clear and more universal right here right now, as it would require less maintenance in future.